orchestrator-salt/src/main/resources/salt/salt/datalake_metrics/calculate_data_sizes/init.sls

```diff
@@ -0,0 +1,19 @@
+/opt/salt/scripts/calculate_data_sizes.sh:
+  file.managed:
+    - makedirs: True
+    - mode: 750
+    - source: salt://sizes/scripts/calculate_data_sizes.sh
+
+/opt/salt/sizes/.pgpass:
```

Comment:
Isn't this renamed to `datalake_metrics/calculate_data_sizes`?
Shouldn't this be updated to the new location?

Reply:
Ah yes I apologize for this Tamas, I meant to modify everything. I will fix this now and then test with a DL.